tamr_client/instance/backup.py

```diff
@@ -0,0 +1,94 @@
+from tamr_client import response
+from tamr_client._types import Instance, Session, URL
+from tamr_client.exception import TamrClientException
+
+
+class AlreadyCanceled(TamrClientException):
```

Comment:
@pcattori @skalish would appreciate thoughts on the name of this exception

Reply:
What is the result of trying to cancel a cancelled backup in the API?  How does it differ from trying to cancel a failed, finished, or nonexistent backup?  Depending on the answers to this I might prefer something like `NotRunning`.

Reply:
Canceling an already canceled backup: ```
{
  "status": 400,
  "class": "javax.ws.rs.BadRequestException",
  "message": "Backup has already completed with status CANCELED. Cancellation failed!",
```

Canceling a failed backup: ```
{
  "status": 400,
  "class": "javax.ws.rs.BadRequestException",
  "message": "Backup has already completed with status FAILED. Cancellation failed!",
```

Canceling a successful/finished backup: ```
{
  "status": 400,
  "class": "javax.ws.rs.BadRequestException",
  "message": "Backup has already completed with status SUCCEEDED. Cancellation failed!",
```

Canceling a nonexisting backup: ```
{
  "status": 404,
  "class": "com.tamr.common.except.ResourceNotFoundException",
  "message": "Invalid backup ID: asfkjhasdlkfjahsd",
```

Reply:
To conform to the language in those messages, maybe `AlreadyCompleted` and pass the message.  I'm not totally sure.

Reply:
Maybe something like `InvalidOperation`? This way it doesn't have to be specific to the cancel task, it can be applied to other types of operations we may run across that are invalid/not possible for other reasons.